website/src/content/docs/start_here/install.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SCAAML Installation
+description: SCAAML software installation
+---
+
+Installation of the SCAAML Python package.
+
+## Dependencies
+
+### [Optional] PicoScope &reg; 6424E
+
+To use the Python oscilloscope control of a [PicoScope
+&reg;](https://www.picotech.com/products/oscilloscope) 6424E one needs to
+install the corresponding libraries.  If you do not plan to use this the only
+downside of no having these libraries is some failing unit-tests.  Follow the
```

Comment:
typo: of not having

s/is some failing unit-tests/is that some unit-tests will fail/